README.md

```diff
@@ -86,10 +84,10 @@ following JSON via a POST request:
 }
 ```
 
-`spanId` is an unique identifier for the span. Define the root span of a trace
+`spanId` is a unique identifier for the span. Define the root span of a trace
```

Comment:
Why is this stuff even here? Its place is https://www.instana.com/docs/api/agent#trace-sdk-web-service. Delete?

Reply:
Beats me. Deleted most of it and added a link to the normal documentation.